crates/reporting/src/error/parse.rs

```diff
@@ -246,6 +246,16 @@ fn to_expr_report<'a>(
                     Context::InNode(Node::WhenBranch, _pos, _) => {
                         return to_unexpected_arrow_report(alloc, lines, filename, *pos, start);
                     }
+
+                    Context::InDef(_pos) => {
+                        vec![alloc.stack([
+                            alloc.concat([alloc.reflow(
+                                "Did you mean to define a function? Condider defining this as",
+                            )]),
+                            alloc.vcat(vec![alloc.text("id = \\x -> x")]),
```

Comment:
we have special style for suggestions, see e.g. 
```rust
                    alloc.reflow("Looks like you are trying to define a function. "),
                    alloc.reflow("In roc, functions are always written as a lambda, like "),
                    alloc.parser_suggestion("increment = \\n -> n + 1"),
                    alloc.reflow("."),
```

`parser_suggestion` will give things some nice colors in the terminal. I think adding a ` .indent(4)` to the suggestion line will also make it look nicer

Reply:
I updated the code and  I used `parser_suggestion`.
I also changed the message with yours which I think is nicer.